CONTRIBUTING.md

```
@@ -1,3 +1,3 @@
 # Contributing
 
-For a detailed guide how to contribute code to this repository, look at the guide [here](https://fdtdx.readthedocs.io/en/latest/contributing.html)
+For a detailed guide how to contribute code to this repository, look at the guide [here](https://fdtdx.readthedocs.io/en/latest/05_contributing.html)
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)